sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/CdnManagementClient.cs

```diff
@@ -59,6 +59,17 @@ public partial class CdnManagementClient : ServiceClient<CdnManagementClient>, I
         /// </summary>
         public string ApiVersion { get; private set; }
 
+        /// <summary>
+        /// Azure Subscription ID.
+        /// </summary>
+        public string SubscriptionId1 { get; set; }
```

Comment:
There's definitely something wrong here. Please make sure the SDK is generated by running `generate.ps1` and your [build tools are up to date.](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools)

Besides, please point to me the link to your openapi spec review PR. Thanks